pkg/build/types/bundle_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package types
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestNewBundle(t *testing.T) {
+	invalidDir := "notExsitingDir/"
+	validDir := "bundleTests/"
+	prefixes := []string{"", "dummyPrefix"}
+	testFalseSections := [][]string{{"dummy1", "dummy2", "none"},
+		{"none"},
+		{"dummy1", "dummy2"}}
+	testTrueSections := [][]string{{"all"},
+		{"dummy", "all"},
+		{"test"},
+		{"dummy", "test"}}
+
+	// We create the test directory
+	err := os.MkdirAll(validDir, 0755)
```

Comment:
please use ioutil.TempDir to create a temporary directory.

Reply:
fixed